Week3/homework/step2-3.js

```diff
@@ -30,10 +31,15 @@ console.log('while', repeatStringNumTimesWithWhile('abc', 3));
 function repeatStringNumTimesWithDoWhile(str, num) {
   // eslint-disable-next-line prefer-const
   let result = '';
-
-  // Replace this comment and the next line with your code
-  console.log(str, num, result);
-
+  let i = 0;
+  if (num <= 0) {
+    result += '';
+  } else {
+    do {
+      i++;
+      result += str;
+    } while (i < num);
+  }
```

Comment:
Any reason why you didn't add the `num <= 0` check to the `for` and `while` loops? 🙂

Reply:
I forgot to set the condition for negative values in both for and while. 
The file has been updated.